framework/src/convergence/DefaultMultiAppFixedPointConvergence.C

```diff
@@ -0,0 +1,214 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "DefaultMultiAppFixedPointConvergence.h"
+#include "FEProblemBase.h"
+#include "FixedPointSolve.h"
+#include "Console.h"
+#include "SteffensenSolve.h"
+
+registerMooseObject("MooseApp", DefaultMultiAppFixedPointConvergence);
+
+InputParameters
+DefaultMultiAppFixedPointConvergence::validParams()
+{
+  InputParameters params = DefaultConvergenceBase::validParams();
+  params += FixedPointSolve::fixedPointDefaultConvergenceParams();
+  params.addClassDescription("Default fixed point convergence criteria.");
```

Comment:
we should trim down the execute_on flags for this object, if not make it private and hardcoded.

most options would be ignored